docs/maps/maps-getting-started.asciidoc

```diff
@@ -93,37 +94,38 @@ image::maps/images/gs_add_cloropeth_layer.png[]
 [[maps-add-elasticsearch-layer]]
 === Adding layers for {es} data
 
-You'll add two layers for {es} data. The first layer displays documents, and the
-second layer displays aggregated data.
-The raw documents appear when you zoom in the map to show smaller regions.
-The aggregated data 
-appears when you zoom out the map to show larger amounts of the globe.
+To avoid overwhelming the user with too much data at once, you'll add two layers for {es} data.
+
+. The first layer will show aggregated data that represents many documents.
+The layer will appear when the user zooms out the map to show larger amounts of the globe.
+. The second layer will display individual documents.
+The layer will appear when the user zooms in the map to show smaller regions.
 
 ==== Add a vector layer from the document source
 
 This layer displays web log documents as points.
-The layer is only visible when you zoom in the map past zoom level 9.
+The layer is only visible when users zoom in the map past zoom level 9.
 
 . In the map legend, click *Add layer*.
 . Click the *Documents* data source.
 . Set *Index pattern* to *kibana_sample_data_logs*.
 . Click the *Add layer* button.
 . Set *Layer name* to `Actual Requests`.
-. Set *Min zoom* to 9 and *Max zoom* to 24.
+. Set *Zoom range for layer visibility* to the range [9, 24].
 . Set *Layer transparency* to 1.
 . Set *Fill color* to *#2200ff*.
 . Click *Save & close*.
```

Comment:
Some suggestions:

- Include an an image of what the map looks like in this zoom range.
- Change the zoom range to 5-24

Reply:
I opted to include a screen shot of the app between zoom levels 9 and 24